onnx/__init__.py

```diff
@@ -150,12 +167,12 @@ def load_model_from_string(s: bytes, format: Optional[Any] = None) -> ModelProto
 
     Returns:
         Loaded in-memory ModelProto
-    '''
+    """
     return _deserialize(s, ModelProto())
 
 
 def load_tensor_from_string(s: bytes, format: Optional[Any] = None) -> TensorProto:
-    '''
+    """
```

Comment:
One thing irrelevant to this PR: is it possible to silent this kind of warning (Undefined variables from ONNX) from pylint? If not, shall we build ONNX first in this CI to let pylint understand ONNX? Building ONNX might be helpful for the next mypy check as well.

Reply:
#4428